deepmd/descriptor/se_ar.py

```diff
@@ -7,8 +7,9 @@
 from deepmd.env import op_module
 from .descriptor import Descriptor
 
+@Descriptor.register("se_ar")
 class DescrptSeAR (Descriptor):
-    def __init__ (self, jdata):
+    def __init__ (self, **jdata):
```

Comment:
This change actually introduces a bug, `DescrptSeAR` because takes `dict` as input. 

`DescrptSeAR` is replace by `DescrptHybrid`. 

Now deepmd-kit does not accept the descriptor type `"se_ar"`, so we can safely remove it

Reply:
Ok, I've reverted changes to DescrptSeAR.

Reply:
Can we directly remove this file?

Reply:
Yes we can. Remember also remove it UT.